webapp/src/components/Metrics/PerformanceAnalysis.tsx

```diff
@@ -184,28 +181,99 @@ const PerformanceAnalysis: React.FC<Props> = ({ jobId, pipeline }) => {
       ...NUMBER_COL_DEF,
       field: "utteranceCount",
       headerName: "Utterance Count",
-      minWidth: 160,
+      minWidth: 150,
     },
     ...ALL_OUTCOMES.map<Column<Row>>((outcome) => ({
       ...NUMBER_COL_DEF,
       field: outcome,
       headerName: OUTCOME_PRETTY_NAMES[outcome],
+      minWidth: 105,
       renderHeader: () => OutcomeIcon({ outcome }),
-      valueGetter: ({ row }) => row.outcomeCount[outcome] / row.utteranceCount,
-      valueFormatter: percentageFormatter,
+      renderCell: ({ row }: GridCellParams<undefined, Row>) =>
+        isNaN(row.outcomeCount[outcome] / row.utteranceCount) ? (
+          "--%"
+        ) : (
+          <Box
+            display="grid"
+            gridAutoColumns={50}
+            gridAutoFlow="column"
+            alignItems="center"
+          >
+            {formatRatioAsPercentageString(
+              (row.outcomeCount[outcome] / row.utteranceCount) as number,
+              1
+            )}
+            <Box
+              component={motion.div}
+              key={outcome}
+              overflow="auto"
+              height="90%"
+              animate={{
+                width: `${
+                  (100 * row.outcomeCount[outcome]) / row.utteranceCount || 1
```

Comment:
I don't think the width should be 1% when the value is 0%. Any reason for that?
```suggestion
                  (100 * row.outcomeCount[outcome]) / row.utteranceCount
```

Reply:
Interesting. Let's keep it, then, and review that with the team. I find it wrong, but it's a very tiny imprecision so it might not matter.

Reply:
I think we can show no bar for 0% and see what the user think of it. what say?

Reply:
I don't have a strong preference either way.